components/21-atoms/table/table-mobile.functions.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+var MobileTable = function (element) { // eslint-disable-line no-unused-vars
+  /**
+   * The table wrapper.
+   */
+  var table = null;
+
+  /**
+   * The table caption.
+   */
+  var caption = null;
+
+  /**
+   * The responsive list version of the table.
+   */
+  var tableList = null;
+
+  /**
+   * Add a wrapper div with class .table-list as next sibling to the table.
+   */
+  var addTableList = function () {
+    var tableListContainer = document.createElement('div');
```

Comment:
Why not 'tableList = document...' and remove line 27.
Confusing naming anyway.

Reply:
Mmm good catch :) You're right. 👍